lib/ci/container.rb

```diff
@@ -1,5 +1,7 @@
 require 'docker'
 
+Docker::API_VERSION = '1.24'.freeze
```

Comment:
This seems folly. If anything it should set the version only if it is not already >=1.24